src/rmp/test/gcd_restructure.tcl

```diff
@@ -10,7 +10,10 @@ report_worst_slack
 
 report_design_area
 
-restructure -target area -abc_logfile results/abc.log
+set tiehi "NangateOpenCellLibrary/LOGIC1_X1/Z"
+set tielo "NangateOpenCellLibrary/LOGIC0_X1/Z"
+
+restructure -target area -abc_logfile results/abc_rcon.log  -tielo_pin $tielo -tiehi_pin $tiehi
```

Comment:
I am not a fan of using pin here. It is what DEF uses (and sdc for that matter) but I consistently use port everywhere outside of required sdc compatibility